frontend/src/components/pages/StudentAssessment.tsx

```diff
@@ -0,0 +1,216 @@
+/* eslint-disable react/prop-types */
+import React from "react";
+import { useHistory } from "react-router-dom";
+import {
+  Box,
+  Text,
+  Stack,
+  Button,
+  Image,
+  VStack,
+  HStack,
+} from "@chakra-ui/react";
+import {
+  JUMP_MATH_LOGO,
+  MULTI_SELECT,
+  MULTI_CHOICE,
+  SHORT_ANSWER,
+} from "../../assets/images";
+import { STUDENT_LANDING } from "../../constants/Routes";
+
+type BoxProps = {
+  children: React.ReactNode;
+  marginTop?: string;
+  marginLeft?: string;
+};
+
+const InnerBox: React.FC<BoxProps> = ({ children, marginTop, marginLeft }) => (
+  <Box
+    backgroundColor="blue.50"
+    borderRadius="10px"
+    position="absolute"
+    padding="10px"
+    gap="40px"
+    width="115px"
+    height="115px"
+    marginTop={marginTop}
+    marginLeft={marginLeft}
+  >
+    {children}
+  </Box>
+);
+
+const OuterBox: React.FC<BoxProps> = ({ children, marginTop, marginLeft }) => (
+  <Box
+    backgroundColor="rgba(232, 237, 241, 0.2)"
+    borderRadius="10px"
+    position="absolute"
+    padding="32px"
+    width="480px"
+    height="376px"
+    marginTop={marginTop}
+    marginLeft={marginLeft}
+  >
+    {children}
+  </Box>
+);
+
+const assessmentMetadata = {
+  numOfQuestions: 12,
+  totalPoints: "50 + 1 (Bonus)",
```

Comment:
```suggestion
  totalPoints: "50",
```
bonus was removed from design